docs-importer/main.js

```diff
@@ -400,6 +442,9 @@ const importFile = (file, remove, resolve) => {
         }
         spinner.succeed('Upload successful for ' + file);
         if (remove) {
+          if (prefs.importer.copyFolder) {
+            fs.copyFileSync(file, prefs.importer.copyFolder + filename);
+          }
           fs.unlinkSync(file);
         }
```

Comment:
This was somewhat right at the beginning, as the tags are removed from the filename
```suggestion
          if (prefs.importer.copyFolder) {
            fs.copyFileSync(file, prefs.importer.copyFolder + file.replace(/^.*[\\\/]/, ''));
            fs.unlinkSync(file);
          }
          else {fs.unlinkSync(file);}
        }
```